syntax = "proto2";

package tec.proto;

import "components.proto";

message GameStateUpdate {
	required uint64 state_id = 1;
	required uint64 command_id = 2;
	repeated Entity entity = 3;
}